decorator/timer.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+def decorator(function):
```

Comment:
Ważne, żeby nazwy funkcji były tak self-explanatory, jak to możliwe, odzwierciedlając to, do czego służą, więc tę funkcję dekorującą najlepiej byłoby chyba nazwać po prostu `timer`.